app/controllers/users_controller.rb

```diff
@@ -85,11 +84,24 @@ def show
       render_404
     else
       respond_to do |format|
-        format.html { render layout: 'no_menu' }
+        format.html do
+          show_respond_html(events)
+        end
```

Comment:
We don't really need the `respond_to` block any more. There is only one entry in it so for the client, it should not make a difference at all. 
We could also add a file-wide `respond_to :html` to the top of the file to have it clarified for all methods.

Reply:
Good suggestion. Fixed.